sentry_sdk/integrations/tornado.py

```diff
@@ -164,7 +164,7 @@ def tornado_processor(event, hint):
 
         with capture_internal_exceptions():
             method = getattr(handler, handler.request.method.lower())
-            event["transaction"] = transaction_from_function(method)
+            event["transaction"] = transaction_from_function(method) or ""
```

Comment:
I checked quickly in we never check for `None` anywhere so I guess this is OK.

Reply:
Yeah, `event["transaction"]` has a type of `str`, so if we set it, we need to set it to a string. `transaction_from_function` can return `None` (although I think it is rare), hence why I added the `or ""`.